alacritty/src/event.rs

```diff
@@ -816,36 +818,49 @@ impl<'a, N: Notify + 'a, T: EventListener> input::ActionContext<T> for ActionCon
         }
     }
 
-    /// Paste a text into the terminal.
-    fn paste(&mut self, text: &str, bracketed: bool) {
+    /// Write text into the terminal or widget.
+    fn write_str(&mut self, text: &str, mode: WriteStrMode) {
```

Comment:
I was thinking maybe `write_user_input` could be a good name? Since this is specifically about handling text input by the user rather than writing arbitrary strings like escape sequences.

Reply:
yeah, but the intention is to write strings, and for regular input we have different API... Maybe `write_user_text`.

Reply:
The intention isn't to write arbitrary strings though. You wouldn't want arbitrary strings to go through bracketed paste.